guides/plugins/plugins/plugin-fundamentals/logging.md

```diff
@@ -0,0 +1,92 @@
+# Logging
+
+## Overview
+
+As a plugin developer, you may want to log certain actions or errors to a log file to aid in debugging or to simply keep a record of performed actions.
+
+## Prerequisites
+
+This guide is built upon our [plugin base guide](../plugin-base-guide.md), which explains the basics of a plugin as a whole. Make sure to have a look at it to get started on building your first plugin.
+
+## Configuring Monolog
+
+First, you must make sure that your plugin loads package configuration from the `/Resources/config/packages` folder:
+
+{% code title="<plugin root>/src/SwagBasicExample.php" %}
```

Comment:
Please use [code groups](https://vitepress.dev/guide/markdown#code-groups) - old Gitbook syntax is not supported anymore.

Reply:
nice thanks, will update

Reply:
Also done!